build.yaml

```diff
@@ -198,6 +198,20 @@ steps:
     image:
       valueFrom: create_certs_image.image
     script: |
+
+      {% if not deploy %}
+      {% raw %}
+      kubectl get secret ssl-config-hail-root --template='{{index .data "hail-root-cert.pem"}}' \
+        | base64 --decode \
+        | openssl x509 -checkend 0 -noout -in -
+      {% endraw %}
+
+      if [ "$?" -ne 0 ]
+      then
+          kubectl delete secret -n {{ default_ns.name }} ssl-config-hail-root
+      fi
```

Comment:
This is unrelated updates of the root cert, yeah?

Reply:
Ya this was merged in a different PR already. Dropped this commit.